### DISCUSSION TITLE
Missing libcudnn_ops_infer.so.8 in cuda version of the docker  git tag bc29d5d , as a result stt is ko

### DISCUSSION THREAD

**Discussion by ptempier** [open-webui/open-webui#6071] [General]
Please tell if you need more details.

My understanding is that driver and cuda version come from the host 
but cudnn is a library and has to come from the container

cudnn v8 requires cuda 11.8, but the dockerfile use cuda 12, not sure while the file is missing.

stt works fine if i dont use open-webui cuda version

On the host :
NVIDIA-SMI 560.35.03              Driver Version: 560.35.03      CUDA Version: 12.6 

docker -v
Docker version 27.3.1, build ce12230

container deployed with these options :
    deploy:
      resources:
        reservations:
          devices:
            - driver: nvidia
              count: 4
              capabilities: [gpu]

INFO  [open_webui.env] 'AUDIO_TTS_MODEL' loaded from the latest database entry
INFO  [open_webui.env] 'AUDIO_TTS_VOICE' loaded from the latest database entry
WARNI [langchain_community.utils.user_agent] USER_AGENT environment variable not set, consider setting it to identify your requests.
INFO  [open_webui.apps.audio.main] whisper_device_type: cuda
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
INFO  [open_webui.apps.openai.main] get_all_models()
INFO  [open_webui.apps.ollama.main] get_all_models()
INFO  [open_webui.apps.audio.main] file.content_type: audio/wav

  ___                    __        __   _     _   _ ___ 
 / _ \ _ __   ___ _ __   \ \      / /__| |__ | | | |_ _|
| | | | '_ \ / _ \ '_ \   \ \ /\ / / _ \ '_ \| | | || | 
| |_| | |_) |  __/ | | |   \ V  V /  __/ |_) | |_| || | 
 \___/| .__/ \___|_| |_|    \_/\_/ \___|_.__/ \___/|___|
      |_|                                               

      
v0.3.32 - building the best open-source AI user interface.

https://github.com/open-webui/open-webui

transcribe /app/backend/data/cache/audio/transcriptions/xxxx.wav
INFO:     xxx2:0 - "GET /ws/socket.io/?EIO=4&transport=polling&t=P9numMY HTTP/1.1" 200 OK
INFO:     ('xxx', 0) - "WebSocket /ws/socket.io/?EIO=4&transport=websocket&sid=xxx" [accepted]
INFO:     connection open
INFO:     xxx:0 - "GET /ws/socket.io/?EIO=4&transport=polling&t=xxxx HTTP/1.1" 200 OK
INFO:     xxx:0 - "POST /ws/socket.io/?EIO=4&transport=polling&t=xxx HTTP/1.1" 200 OK
Could not load library libcudnn_ops_infer.so.8. Error: libcudnn_ops_infer.so.8: cannot open shared object file: No such file or directory
CUDA is enabled, appending LD_LIBRARY_PATH to include torch/cudnn & cublas libraries.
/app/backend/open_webui
/app/backend

**Comment by justinh-rahb** [comment]
Been reportedly fixed as of 0.3.33
